prisma/migrations/20240226024040_init/migration.sql

```diff
@@ -0,0 +1,93 @@
+-- CreateEnum
+CREATE TYPE "preeshHeaviness" AS ENUM ('LOWKEY', 'HEAVY', 'HEAVYFR', 'SUPERPREESH');
+
+-- CreateTable
+CREATE TABLE "Beast" (
+    "id" SERIAL NOT NULL,
+    "gamerTag" VARCHAR(20) NOT NULL,
+    "email" TEXT NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL,
+
+    CONSTRAINT "Beast_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "Preesh" (
+    "id" SERIAL NOT NULL,
+    "text" TEXT NOT NULL,
+    "authorId" INTEGER NOT NULL,
+    "receiverId" INTEGER NOT NULL,
+    "praysh" BOOLEAN NOT NULL DEFAULT false,
+    "heaviness" "preeshHeaviness" NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL,
+
+    CONSTRAINT "Preesh_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "Comment" (
+    "id" SERIAL NOT NULL,
+    "text" TEXT NOT NULL,
+    "preeshId" INTEGER NOT NULL,
+    "authorId" INTEGER NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL,
+
+    CONSTRAINT "Comment_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "_BeastieBros" (
+    "A" INTEGER NOT NULL,
+    "B" INTEGER NOT NULL
+);
+
+-- CreateTable
+CREATE TABLE "_prees" (
+    "A" INTEGER NOT NULL,
+    "B" INTEGER NOT NULL
+);
```

Comment:
Are __BeastieBros and __prees placeholder tables for now? If not I would use more descriptive attribute names.